app/views/static_pages/home.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="jumbotron jumbotron-fluid">
```

Comment:
Check this out to convert your erbs to hamls 

https://github.com/haml/haml-rails#converting-rails-application-layout-file-to-haml-format

Reply:
Thank you! Haml has been giving me some difficulty. I was trying to right it as hail to start with but I'm not comfortable enough with it yet and it was taking too much time for me to figure out. So I was writing it all out in erb first then switch it to Haml when I finish. :)